### DISCUSSION TITLE
Trip Calculation Formula

### DISCUSSION THREAD

**Discussion by Hayden-Badenhop** [CarolineChief/Tafe240261#9] [General]
I know its a long shot, but does anyone know the calculation formula for the trip-feature as its not on the appendix? 

Thanks Hayden

**Comment by CarolineChiefari** [comment]
Hey Hayden,
Checking on the ASI, it seems the trip calculator was meant to just be a 'coming soon' feature for the assignment, but because we were a team of 3, you got hit with it for real. 
Best bet is to jump into Stephens IT Prac tomorrow (Tuesday 9-11am) and ask him directly or email him. Here's the link to all the ITPracs https://learn.tafesa.edu.au/mod/lti/view.php?id=714126&forceview=1.
Caroline
